src/Private/FileWatchers.ps1

```diff
@@ -54,11 +54,7 @@ function Start-PodeFileWatcherRunspace {
     $watchScript = {
         param(
             [Parameter(Mandatory = $true)]
-            $Watcher,
-
-            [Parameter(Mandatory = $true)]
-            [int]
-            $ThreadId
```

Comment:
The parameter needs to stay, plus the property from `Add-PodeRunspace -Parameters`, as I know of some people who do utilise this in their scripts to see which "thread" their script is being invoked from, and even `Write-PodeErrorLog` uses it 😃. Same in:

* PodeServer.ps1
* SmtpServer.ps1
* TcpServer.ps1
* Private/WebSockets.ps1

Reply:
I removed them because I thought that I added them to make the naming work.
I'm going to rollback them 😣